contracts/lib/MutualUpgradeV2.sol

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Commenting the diff **[here](https://gist.github.com/bweick/79c26c48c6c6e7cf7db8e3cb3e7e5bf2)**